src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -560,35 +594,45 @@ function updateConstSumMessageQn(qnNum) {
                         fillWithZeroIfEmpty($inputFieldElement);
                     }
                 }
+            } else if (remainingPoints > 0) {
+                message += `${errorIcon} Actual total is ${sum}! Distribute the remaining ${remainingPoints} points.<br>`;
+                messageElement.addClass('text-color-red');
+                messageElement.removeClass('text-color-blue');
+                messageElement.removeClass('text-color-green');
+            } else {
+                message += `${errorIcon} Actual total is ${sum}! Remove the extra ${-remainingPoints} points allocated.<br>`;
+                messageElement.addClass('text-color-red');
+                messageElement.removeClass('text-color-blue');
+                messageElement.removeClass('text-color-green');
             }
-        } else if (remainingPoints > 0) {
-            message = `${remainingPoints} points left to distribute.`;
-            messageElement.addClass('text-color-red');
-            messageElement.removeClass('text-color-green');
-            messageElement.removeClass('text-color-blue');
-        } else {
-            message = `Over allocated ${-remainingPoints} points.`;
-            messageElement.addClass('text-color-red');
-            messageElement.removeClass('text-color-green');
-            messageElement.removeClass('text-color-blue');
-        }
-
-        if (!allNotNumbers && forceSomeUnevenDistribution && allSame && numOptions !== 1) {
-            message += ' Different amount of points should be given to some options.';
-            messageElement.addClass('text-color-red');
-            messageElement.removeClass('text-color-green');
-        }
 
-        if (!allNotNumbers && ((forceUnevenDistribution && !forceSomeUnevenDistribution) || forceAllUnevenDistribution)
-                && !allUnique) {
-            message += ' The same amount of points should not be given multiple times.';
-            messageElement.addClass('text-color-red');
-            messageElement.removeClass('text-color-green');
+            if (forceSomeUnevenDistribution) {
+                if (allSame && numOptions !== 1) {
```

Comment:
forgot to say that i like how u separated this part of logic into nested if-else blocks. It looks simpler now.